WooCommerce/Classes/ViewRelated/ReusableViews/Section Headers/PlainTextSectionHeaderView.swift

```diff
@@ -0,0 +1,31 @@
+import UIKit
+
+/// `UITableView` section header view that displays a text label.
+/// The `label: UILabel` property can be configured by the consumer where it is used.
+final class PlainTextSectionHeaderView: UITableViewHeaderFooterView {
+    private(set) lazy var label: UILabel = UILabel()
+
+    override init(reuseIdentifier: String?) {
+        super.init(reuseIdentifier: reuseIdentifier)
+        configureBackground()
+        configureLabel()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+private extension PlainTextSectionHeaderView {
+    func configureBackground() {
+        contentView.backgroundColor = .basicBackground
+    }
+
+    func configureLabel() {
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 0
+
+        contentView.addSubview(label)
+        contentView.pinSubviewToAllEdges(label, insets: .init(top: 16, left: 16, bottom: 16, right: 16))
```

Comment:
nit: should we move this to a `Constants` enum?

Reply:
yup! updated in https://github.com/woocommerce/woocommerce-ios/pull/3306/commits/9bd8ddd9b7ba9447f064a10049b4bca1cdde5a17